CMakeLists.txt

```diff
@@ -44,7 +44,6 @@ option(AMBER_SKIP_SAMPLES
 option(AMBER_USE_LOCAL_VULKAN "Build with vulkan in third_party" OFF)
 option(AMBER_ENABLE_SWIFTSHADER
   "Build using SwiftShader" ${AMBER_ENABLE_SWIFTSHADER})
-option(AMBER_USE_LOCAL_VULKAN "Build with vulkan in third_party" OFF)
```

Comment:
This is redundant with line 44

Reply:
Yea, I think it was a bad merge at one point. I've also removed it in my DXC CL, so we're going to conflict.